util/perl/TLSProxy/Proxy.pm

```diff
@@ -606,6 +618,39 @@ sub clientstart
     return $success;
 }
 
+sub construct_alert_message
+{
+    my ($self) = @_;
```

Comment:
Since TLS is not supported here you might do a 
```
die "construct_alert_message does not support TLS" if !self->isdtls;
```
Code sample most likely does not run, please update it.